currency-converter/src/components/Dropdown.tsx

```diff
@@ -0,0 +1,71 @@
+import Select, { StylesConfig } from 'react-select';
+
+interface IDropdown {
+  label: string;
+}
+
+export default function Dropdown({ label }: IDropdown) {
+  const currencies = [
```

Comment:
this should be declared in parent component, and should be passed as prop to this component.